query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/filters/filter_regression.rs

```diff
@@ -65,10 +65,22 @@ mod fr_one_to_m {
           @r###"{"data":{"findManyCompany":[{"id":135}]}}"###
         );
 
-        insta::assert_snapshot!(
-          run_query!(&runner, r#"query { findManyLocation(where: { company: { is: { id: { equals: 135 }}}}){ id }}"#),
-          @r###"{"data":{"findManyLocation":[{"id":311},{"id":314}]}}"###
-        );
+        match runner.connector_version() {
+            ConnectorVersion::TiDB => is_one_of!(
+                run_query!(
+                    &runner,
+                    r#"query { findManyLocation(where: { company: { is: { id: { equals: 135 }}}}){ id }}"#
+                ),
+                vec![
+                    r#"{"data":{"findManyLocation":[{"id":311},{"id":314}]}}"#,
+                    r#"{"data":{"findManyLocation":[{"id":314},{"id":311}]}}"#
```

Comment:
No guaranteed ordering here?

Reply:
Unfortunately, if the SQL statement does not clearly declare the `ORDER BY`, TiDB does not guarantee the order of returning results.

> From @windtalker:
>
> TiDB is a distributed database, if the sql does not require result order, it is normal that the order of result is not stable(We can not sync the execution between different nodes)

Related issue: https://github.com/pingcap/tidb/issues/37285

Reply:
Us testing the order without clearly specifying the order is a bit of a smell - so maybe we should actually adapt the test.

Reply:
Sorry bit lost what is the recommendation here, should the tests include order by clause or should this test fail with some reasoning ?

Reply:
I can not say that yet - for now we are just collecting information, to then later make decisions on each one when we have an overview and then can go through all the cases one by one.